python/RDS_LOGGING_ENABLED/RDS_LOGGING_ENABLED.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2017-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may
+# not use this file except in compliance with the License. A copy of the License is located at
+#
+#        http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for
+# the specific language governing permissions and limitations under the License.
+
+'''
+#####################################
+##           Gherkin               ##
+#####################################
+Rule Name:
+  RDS_LOGGING_ENABLED
+
+Description:
+  check whether respective logs are enabled or not.
+      For Oracle engine -- ["trace", "audit", "alert", "listener"]
+      For Postgres engine -- ["postgresql", "upgrade"]
+      For MariaDB -- ["audit", "error", "general", "slowquery"]
+      For Mysql -- ["audit", "error", "general", "slowquery"]
+      For SqlServer -- ["error", "agent"]
+
+Indicative Severity:
+  Medium
+
+Trigger:
+  Periodic
+
+Reports on:
+  AWS::RDS::DBInstance
+
+Rule Parameters:
+  None
+
+Scenarios:
+  Scenario: 1
```

Comment:
Add scenario 1:
Given: there is no RDS instance (non-Aurora)
Then: return Not applicable